modal/cli/import_refs.py

```diff
@@ -110,7 +110,7 @@ def get_by_object_path(obj: Any, obj_path: str) -> Optional[Any]:
 def _infer_function_or_help(
     app: App, module, accept_local_entrypoint: bool, accept_webhook: bool
 ) -> Union[Function, LocalEntrypoint]:
-    function_choices = set(tag for tag, func in app.registered_functions.items() if not func.info.is_service_class())
+    function_choices = set(tag for tag in app.registered_functions.keys())
```

Comment:
Nit (not worth fixing just FYI) can do this as `set(app.registered_functions)` or just do set operations on the `.keys()` directly.